src/ae.c

```diff
@@ -345,6 +345,13 @@ static int processTimeEvents(aeEventLoop *eventLoop) {
     return processed;
 }
 
+/* This function provides direct access to the aeApiPoll call.
+ * It is intended to be called from a custom poll function,
+ * currently invoked from IOThreadPoll.*/
```

Comment:
```suggestion
 * It is intended to be called from a custom poll function. */
```
Since ae.c is supposed to be generic, and also these references fall out of date.

Reply:
Done